runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -102,10 +114,33 @@ public void onScriptCallbackEvent(ScriptCallbackEvent e)
 						continue;
 					}
 
-					level += Experience.getLevelForXp(client.getSkillExperience(s));
+					if (config.alterTotalLevel())
+					{
+						level += Experience.getLevelForXp(client.getSkillExperience(s));
+					}
+					else
+					{
+						if (Experience.getLevelForXp(client.getSkillExperience(s)) > 99)
```

Comment:
This whole if/else statement can be replaced with `level += Math.min(skillLevel, 99);`